resolwe_bio/kb/tests/test_feature.py

```diff
@@ -192,6 +193,14 @@ def test_feature_search(self):
         )
         self.assertEqual(len(response.data), 1)
 
+    def test_serialization(self):
+        FEATURE_SEARCH_URL = reverse("resolwebio-api:kb_feature_search")
+
+        response = self.client.get(FEATURE_SEARCH_URL, {"limit": 1}, format="json")
+        self.assertEqual(len(response.data["results"]), 1)
+        for field_name in FeatureSerializer.Meta.fields:
```

Comment:
This only tests fundamentals of the serializer - that it includes fields specified in `Meta`. It would be better to check that values are set correctly.

I also prefer the WET approach instead of DRY for testing, so it would be better to type actual field names instead of using the `Meta` class.

Reply:
@dblenkus I've modified my approach, please review again.